### ISSUE TITLE
FALSE Alarm - Avira Antivirus Professional Warns about virus in 

### ISSUE THREAD

**Issue by ConEmu** [ConEmu/old-issues#555]
_From [Cuchuk.S...@gmail.com](https://code.google.com/u/101200228523943764078/) on May 25, 2012 11:11:59_

OS version: Win7 SP1 x64

I've attached picture.

**Attachment:** [AviraGoesCrazy.png](http://code.google.com/p/conemu-maximus5/issues/detail?id=555)

_Original issue: http://code.google.com/p/conemu-maximus5/issues/detail?id=555_

**Comment by ConEmu** [comment]
_From [Cuchuk.S...@gmail.com](https://code.google.com/u/101200228523943764078/) on May 25, 2012 11:18:31_

Check of ConEmuSetup.120417.exe succeeds.

**Comment by ConEmu** [comment]
_From [ConEmu.M...@gmail.com](https://code.google.com/u/116287257743594353797/) on May 25, 2012 11:51:16_

Check unpacked 7z of these versions

**Comment by ConEmu** [comment]
_From [Cuchuk.S...@gmail.com](https://code.google.com/u/101200228523943764078/) on May 25, 2012 12:59:56_

Thank you! That helped. Avira didn't showed any messages.

**Comment by ConEmu** [comment]
_From [ConEmu.M...@gmail.com](https://code.google.com/u/116287257743594353797/) on May 25, 2012 13:15:29_

I can't do anything with false alarms of Avira.
Probaly, You, as user of Avira, may contact with their tech support, and ask them, what exactly seems suspicious from their point of view. In this case, may be...

My installer was not significantly changed (only one visual bug in dialog was fixed on 09.05.2012).

BTW, did you import my certificate? https://code.google.com/p/conemu-maximus5/wiki/Certificate

**Comment by ConEmu** [comment]
_From [Cuchuk.S...@gmail.com](https://code.google.com/u/101200228523943764078/) on May 25, 2012 13:56:31_

i will send them setup and ask to remove false alarm.

**Comment by ConEmu** [comment]
_From [ConEmu.M...@gmail.com](https://code.google.com/u/116287257743594353797/) on May 25, 2012 14:02:01_

OK.

KIS have "cloud protection". May be Avira has something like that too.

**Status:** Fixed

**Comment by ConEmu** [comment]
_From [Cuchuk.S...@gmail.com](https://code.google.com/u/101200228523943764078/) on May 25, 2012 14:13:34_

i will post what they will answer (file analyzes takes about 2 weeks).

**Comment by ConEmu** [comment]
_From [Cuchuk.S...@gmail.com](https://code.google.com/u/101200228523943764078/) on June 05, 2012 11:48:06_

Thank you for your recent inquiry.

The present file is falsely detected by Avira AntiVir as TR/Dropper.Gen.

This is a false positive detection, which will be fixed with one of the next VDF updates from AntiVir. With this update, the file itself should not be detected anymore.

Please note that this is a generic false positive.

This means, that this false positive appears on the basis of certain unique characteristics inside the file. Therefore it is possible that similar files will also be reported with this detection.

This false positive will finally be fixed with the next engine update.

Thanks in advance.

For further questions don't hesitate to contact us.
-\- 
Freundliche Gruesse / Best regards
Avira Operations GmbH & Co. KG

Bernd Kersten
Consumer Services - International Services & Support

Avira Operations GmbH & Co. KG
Kaplaneiweg 1, D-88069 Tettnang, Germany
Internet: http://www.avira.com Geschaeftsfuehrender Gesellschafter: Tjark Auerbach
Sitz der Gesellschaft: Tettnang; AG Ulm HRA 722586
----------------------------------------------------------------------
ALLGEMEINE GESCHAEFTSBEDINGUNGEN
Es gelten unsere Allgemeinen Geschaeftsbedingungen (AGB).
Sie finden sie in der jeweils gueltigen Fassung unter:

**Comment by ConEmu** [comment]
_From [ConEmu.M...@gmail.com](https://code.google.com/u/116287257743594353797/) on June 05, 2012 12:09:48_

Well, okay.

**Comment by ConEmu** [comment]
_From [ConEmu.M...@gmail.com](https://code.google.com/u/116287257743594353797/) on June 13, 2012 22:29:23_

Issue 596 has been merged into this issue.

**Comment by ConEmu** [comment]
_From [ConEmu.M...@gmail.com](https://code.google.com/u/116287257743594353797/) on June 14, 2012 13:02:53_

Issue 597 has been merged into this issue.

**Comment by ConEmu** [comment]
_From [ConEmu.M...@gmail.com](https://code.google.com/u/116287257743594353797/) on June 14, 2012 13:06:53_

**Status:** Visible

**Comment by ConEmu** [comment]
_From [ConEmu.M...@gmail.com](https://code.google.com/u/116287257743594353797/) on June 14, 2012 13:07:52_

**Summary:** FALSE Alarm - Avira Antivirus Professional Warns about virus in ConEmuSetup.120524a.exe

**Comment by ConEmu** [comment]
_From [ConEmu.M...@gmail.com](https://code.google.com/u/116287257743594353797/) on November 16, 2013 06:58:22_

Issue 1343 has been merged into this issue.

**Comment by ConEmu** [comment]
_From [ConEmu.M...@gmail.com](https://code.google.com/u/116287257743594353797/) on November 16, 2013 06:58:36_

Issue 1343 has been merged into this issue.

**Comment by ConEmu** [comment]
_From [ConEmu.M...@gmail.com](https://code.google.com/u/116287257743594353797/) on November 16, 2013 06:59:03_

**Labels:** False-Alarm

**Comment by ConEmu** [comment]
_From [ConEmu.M...@gmail.com](https://code.google.com/u/116287257743594353797/) on November 16, 2013 06:59:58_

Issue 596 has been merged into this issue.

**Comment by ConEmu** [comment]
_From [ConEmu.M...@gmail.com](https://code.google.com/u/116287257743594353797/) on November 16, 2013 07:00:28_

Issue 597 has been merged into this issue.

**Comment by ConEmu** [comment]
_From [ConEmu.M...@gmail.com](https://code.google.com/u/116287257743594353797/) on November 16, 2013 07:01:19_

**Summary:** FALSE Alarm - Avira Antivirus Professional Warns about virus in   <span class="oldvalue"> (was: FALSE Alarm - Avira Antivirus Professional Warns about virus in ConEmuSetup.120524a.exe) </span>

**Comment by ConEmu** [comment]
_From [pa...@vdevices.com](https://code.google.com/u/113785944584621865112/) on November 17, 2013 10:28:12_

I'm encountering the same problem. Has anyone found a fix to this?

**Comment by ConEmu** [comment]
_From [ConEmu.M...@gmail.com](https://code.google.com/u/116287257743594353797/) on November 17, 2013 10:53:16_

Report false alarms to Avira.

Many ConEmu console-related features requires Windows API hooking. ConEmuHk wiki describes that. It's strongly not recommended totally disable them (that may cause problems), but if that is only way in your case - "howto" described in ConEmuHk#Conclusion.

**Comment by ConEmu** [comment]
_From [Cuchuk.S...@gmail.com](https://code.google.com/u/101200228523943764078/) on November 17, 2013 11:18:52_

Yes, report version and false alarm to Avira.
They will check version and add file hash to safe list.
Also be on stable branches to not do this very oftern.

**Comment by ConEmu** [comment]
_From [Cuchuk.S...@gmail.com](https://code.google.com/u/101200228523943764078/) on November 17, 2013 11:23:57_

As for hooking i suggest to extract it to separate library (which will update extremely rarely) to make Avira only once check it and add it to safe list its hash.

**Comment by ConEmu** [comment]
_From [ConEmu.M...@gmail.com](https://code.google.com/u/116287257743594353797/) on November 17, 2013 11:26:25_

Are you sure, Avira checks only dll hash? I suppose it check executable too, no?

**Comment by ConEmu** [comment]
_From [ConEmu.M...@gmail.com](https://code.google.com/u/116287257743594353797/) on November 17, 2013 11:27:45_

I see here ConEmu C.exe, but not a library https://conemu-maximus5.googlecode.com/issues/attachment?aid=13430000000&name=conemu+false+positive.png&token=imBcncP3wLdUjdn9b2VKBtp-hIk%3A1384716391237&inline=1

**Comment by ConEmu** [comment]
_From [Cuchuk.S...@gmail.com](https://code.google.com/u/101200228523943764078/) on November 17, 2013 11:27:58_

i wish i could know for sure.

**Comment by ConEmu** [comment]
_From [Cuchuk.S...@gmail.com](https://code.google.com/u/101200228523943764078/) on November 17, 2013 11:44:55_

> I see here ConEmu C.exe, but not a librar
Sure, process name should be shown to user for such things. Dll name will tell user nothing.
But i still don't know how the process is organized.

**Comment by ConEmu** [comment]
_From [Cuchuk.S...@gmail.com](https://code.google.com/u/101200228523943764078/) on November 17, 2013 11:46:48_

probably you right, adding dll to safe list is bad, because malicious software could use it.
